FileSpecs/CharData.md

```diff
@@ -34,6 +34,8 @@ Offset | Type | Name | Description
 001D | Byte | Number of used fingers? |
 0022 | Byte[9] | Item amounts | For equipped items (0 = none)
 002B | Byte[12] | Item amounts | For inventory items (0 = none)
+0037 | Byte | Languages | See below
+003D | Byte | Quest Completion Flag | ID of a flag that toggles which of the two NPC interaction blocks to use, see "NPC Interactions" below
```

Comment:
What do you mean by "ID of a flag"?

Reply:
While I don't know how it is represented, semantically it is equivalent to an index into an array of booleans that stores the quest completion state that applies to this character.

Most NPCs (that have interactions) have their own "quest completion flag" that determines whether the first set or the second set of NPC interactions are available.  However, there are two pairs of NPCs that share these flags, where completing the quest for one enables new interactions with both of the members of that pair.

Do you think that the explanation under "NPC Interactions" needs expansion to clarify this, or do you have a suggestion for describing this more clearly?

Reply:
My biggest question would be where those booleans are stored. I would suspect inside the savegame data like the "global vars" in Ambermoon in Party_data.sav. If you find it, please add a link here for more clarity. Until then it's fine this way.